dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -562,89 +709,117 @@ public void setup() throws Exception {
-                replaceFeature = alwaysAddFeature;
+        locationsChecked.add(serverFolder);
+
+        // Go through all the servers
+        for (String serverName : servers) {
+            Set<File> optionsFound = findFile(new File(pathToAutoFVTTestServers + serverName), "jvm.options");
+            // If options file doesn't exist
+            if (optionsFound.isEmpty()) {
+                // Create it
+                File jvmOptionsCreated = new File(pathToAutoFVTTestServers + serverName + "/jvm.options");
+                if (jvmOptionsCreated.createNewFile()) {
+                    Log.info(c, m, "Successfully created jvm.options in: " + serverName);
+                    optionFilesCreated.add(jvmOptionsCreated);
+                } else
+                    Log.info(c, m, "Failed to create jvm.options in: " + serverName);
+                optionsFound.add(jvmOptionsCreated);
             }
+            serverOptions.addAll(optionsFound);
         }
-        if (replaceFeature != null) {
-            Log.info(c, methodName, "Replace feature [ " + originalFeature + " ] with [ " + replaceFeature + " ]");
-            return replaceFeature;
-        }
-        // We need to check that the feature passed is an EE7/EE8 feature which could have a name change on EE9 that doesn't match
-        // the original feature name it replaces. We also check vice versa if the feature is an EE9 feature that involves a name change
-        // to update from EE9 to EE7/EE8
-        Log.info(c, methodName, "No feature replacement found for [ " + originalFeature + " ]. Verifying if feature name was changed on EE9.");
-        // Reset base feature to not include the "-"
-        baseFeature = baseFeature.substring(0, dashOffset);
-        String featureReplacement = null;
-        for (Map.Entry<String, String> nameChangeEntry : featuresWithNameChangeOnEE9.entrySet()) {
-            if (nameChangeEntry.getValue().equalsIgnoreCase(baseFeature)) {
-                featureReplacement = nameChangeEntry.getKey();
-                Log.info(c, methodName, "Replace EE9 feature [ " + baseFeature + " ] with feature [ " + featureReplacement + " ]");
-                baseFeature = featureReplacement;
-                break;
-            }
-            if (nameChangeEntry.getKey().equalsIgnoreCase(baseFeature)) {
-                featureReplacement = nameChangeEntry.getValue();
-                Log.info(c, methodName, "Replace base feature [ " + baseFeature + " ] with EE9 feature [ " + featureReplacement + " ]");
-                baseFeature = featureReplacement;
-                break;
+
+        Set<File> clientOptions = new HashSet<>();
+        Set<String> clients = new HashSet<>(); // All clients found
+        File clientFolder = new File(pathToAutoFVTTestClients);
+        // Find all jvm.options in the clients
+        clientOptions.addAll(findFile(clientFolder, "jvm.options"));
```

Comment:
Not sure why you have this here.  It is different than what you did for the server side where you only do what you have below.

Reply:
I based this similar to the feature replacement [here](https://github.com/OpenLiberty/open-liberty/pull/20906/files#diff-c9f27c5428b8014664069c8423b6dd0952e3d2329ed7f79958eca90a38f710efR604) I thought we could have loose jvm.options files in the client folder but I'm not exactly familiar on how the files can be organized there so this could possibly be unnecessary. Do you think it can be removed?